lib/pymedphys/_experimental/streamlit/apps/iview_to_dicom.py

```diff
@@ -72,19 +144,60 @@ def _create_portal_image_dicom_dataset(
 
     pixel_spacing = 1 / pixels_per_mm * sid / sad
 
+    date = datetime.strftime("%Y%m%d")
+    time = datetime.strftime("%H%M%S.%f")
+
+    # Many of the below was input without a full understanding of the
+    # implications of the choices being made.
     ds = _pp_dcm_create.dicom_dataset_from_dict(
         {
-            "ImageType": ["ORIGINAL", "PRIMARY", "PORTAL"],
+            "Modality": "RTIMAGE",
+            "InstanceCreatorUID": PYMEDPHYS_ROOT_UID,
+            "SOPClassUID": "1.2.840.10008.5.1.4.1.1.481.1",
+            "SOPInstanceUID": pydicom.uid.generate_uid(prefix=f"{PYMEDPHYS_ROOT_UID}."),
```

Comment:
Study and Series uids also.

Reply:
Does the following address your concerns:

https://github.com/pymedphys/pymedphys/blob/ba2d8dc483b88e1a13d59623847f4038a3455127/lib/pymedphys/_experimental/streamlit/apps/iview_to_dicom.py#L165-L168

The same logic applies for series.

For now, it appears these files do import into pips. I don't want to create study or series UIDs at the moment. In the future I imagine these sorts of things could be passed in as a parameter to a more general function, and then that more general function could be called by a "study" or "series" creator function.

But... that's not anywhere within my scope at the moment.